### ISSUE TITLE
modulefixer misses required module `java.compiler`

### ISSUE THREAD

**Issue by sormuras** [forax/pro#31]
`[modulefixer] fix junit.platform.console.standalone` succeeds with current [SNAPSHOT](https://github.com/junit-team/junit5/commit/a212f1794613291f7a13879ef4fdbf9b783084b7#diff-df5279bb3a9ea062a51956bd0c0af453R26) version. At runtime, it throws:

```
java.lang.IllegalAccessError:
class org.junit.platform.commons.util.PackageUtils
 (in module junit.platform.console.standalone) 
cannot access class javax.lang.model.SourceVersion
 (in module java.compiler)
because module junit.platform.console.standalone
does not read module java.compiler
```

Usage of `javax.lang.model.SourceVersion` should trigger: `requires java.compiler;`

Testable with:
```
set("tester.overrideArguments", list("--select-package", "abc"))
run("tester")
```

**Comment by forax** [comment]
Obviously there is a bug here,
I wonder if it's because the SourceVersion is used inside a method reference (if my bytecode reading fu is correct)

You can workaround it by using modulefixer.additionalRequires
  set("modulefixer.additionalRequires", list(
        "junit.platform.console.standalone=java.compiler"
  ));

**Comment by forax** [comment]
fixed by b1dd0c6e58d485b5afcae0f9ec42039443431888
